### PR TITLE
[AD-496] Support for left and right string functions

### PR DESCRIPTION
**Pull request by andiemontoyeah**
### Summary

- Add support for LEFT(string, length) and RIGHT(string, length)

### Description

- Added support for LEFT and RIGHT in StringFunctionTranslator and added unit tests 
- Updated Tableau connector 
- Updated SQL support documentation

### Related Issue
https://bitquill.atlassian.net/browse/AD-495

### Additional Reviewers
@affonsoBQ
@andiem-bq
@alexey-temnikov
@birschick-bq

### CODE REVIEW

File: tableau-connector/src/dialect.tdd [hunk 1]
```diff
@@ -598,6 +598,22 @@
       <argument type='str' />
       <argument type='int' />
     </function>
+    <function group='string' name='FIND' return-type='int'>
```
Comment (andiemontoyeah) [inline tableau-connector/src/dialect.tdd:601]: Support passing float arguments to FIND() and other string functions as required by TDVT. Use X - X%1 to truncate the decimal portion. If FLOOR() is ever implemented can replace with that.

**Review by affonsov — APPROVED** [review]
lgtm

File: tableau-connector/src/dialect.tdd (hunk 1 above)
Comment (birschick-bq) [reply tableau-connector/src/dialect.tdd:601]: I'm not sure MOD is an appropriate operator for real numbers. Can you add a test?

Comment (andiemontoyeah) [reply tableau-connector/src/dialect.tdd:601]: Added test and added task to possibly implement floor and ceiling with this logic https://bitquill.atlassian.net/browse/AD-502

**Review by birschick-bq — APPROVED** [review]

**Review by alexey-temnikov — APPROVED** [review]

**Pull request merged**